install/install.py

```diff
@@ -116,7 +118,46 @@ def finish(error_code):
             Colorizer.normal("[*] Visit https://github.com/owtf/owtf for help ")
         else:
             Colorizer.success("[*] Finished!")
-            Colorizer.info("[*] Start OWTF by running 'cd path/to/pentest/directory; ./path/to/owtf.py'")
+            Colorizer.info("[*] Start OWTF by running 'workon owtf; cd path/to/pentest/directory; ./path/to/owtf.py'")
+
+
+def setup_virtualenv():
+    Colorizer.normal("[*] Seting up virtual environment named owtf...")
+    # sources files and commands
+    source = 'source /usr/local/bin/virtualenvwrapper.sh'
+    setup_env = 'cd $WORKON_HOME; virtualenv -q --always-copy -p %s owtf >/dev/null 2>&1;'\
+        ' source owtf/bin/activate' % sys.executable
+    dump = sys.executable+' -c "import os, json;print json.dumps(dict(os.environ))"'
+
+    pipe = subprocess.Popen(['/bin/bash', '-c', '%s >/dev/null 2>&1; %s; %s' % (source, setup_env, dump)],
+                            stdout=subprocess.PIPE)
+    env = json.loads(pipe.stdout.read())
+
+    # Update the os environment variable
+    os.environ.update(env)
+
+    if os.path.join(os.environ["WORKON_HOME"], "owtf") == os.environ["VIRTUAL_ENV"]:
+        # Add source to shell config file
+        Colorizer.normal("[*] Adding virtualenvwrapper source to shell config file")
+        shell_rc_path = os.path.join(os.environ["HOME"], ".%src" % os.environ["SHELL"].split(os.sep)[-1])
+        run_command("echo '%s' >> %s" % (source, shell_rc_path))
+    else:
+        Colorizer.warning("Unable to setup virtualenv...")
+
+
+def setup_pip():
+    # Installing pip
+    Colorizer.info("[*] Installing pip")
+    directory = "/tmp/owtf-install/pip/%s" % os.getpid()
+    command = 'command -v pip2 >/dev/null || { wget --user-agent="Mozilla/5.0 (X11; Linux i686; rv:6.0) Gecko/20100101'\
```

Comment:
@delta24 This is important. If by some reason install script was canceled in middle and the pip was not installed. If user runs it again it will skip it. Check other  `distro-independent.cfg` commands which are using `/tmp` directory.